Yoast/wordpress-seo
# Tweak admin redesign

## Context

* Ran through the admin restyling with Kai and Rolf, made tweaks where we seemed fit
* Fixed a couple of issues along the way, missing copy, wrongly styled notifications and a lot of spacing

This is linked to https://github.com/Yoast/javascript/pull/751

### Todo

#### Design

* Content types
  * [x] All setting labels to "hide/show"
  * [x] Archives to separate collapsibles
* Width of the content
  * [x] Increase for larger screens
* Search appearance -> Categories
  * [x] Convert `Category URLs` to a collapsible box
* Tools page
  * [x] Introduce tabs instead of the list
  * [x] Transform the current tabs into collapsibles
* Labels
  * [x] Labels color `#444` needs to be `#303030`
* Buttons
  * [x] Yellow buy buttons font-weight --> 400 (currently 300)
* General feedback
  * [ ] Fix WordPress general alerts alignment
  - [x] Fix toggle labels: “hide/show” instead of “yes/no”
  - [x] Inconsistent font sizes: “you have 1 hidden problem” is on 1.4em, and “you have 1 hidden notification” is on 16px.
  - [x] General > Features > “Upgrade to premium”, should be “Upgrade to Premium”
* Every option page
  * [x] Settings saved layout
* Twitter card
  * [x] Convert to radio buttons
* Schema person/company
  * [x] Convert to radio buttons
* Toggles
  * [x] Make the labels clickable (Yes/No)
* Features page
  * [x] Move the "upsell button" inside the paper

### Mobile design

  * [ ] Walk through and fix tweaking implications

### General finishing up

* [ ] Documentation on this PR: Specify what has been fixed/changed

## Summary


This PR can be summarized in the following changelog entry:

* Visual tweaks to the admin-settings pages

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* Use `yarn link-monorepo` and check out the branch of that PR
* Use `grunt build:js build:css` in the Plugin to build the assets after linking and checking out the branch on the monorepo. 
* Walk through all settings pages and check if they are good enough.
* _Note: Notification center styling is off; needs to be fixed_

## UI changes

## Documentation

## Quality assurance


Fixes #
